jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -90,230 +84,288 @@
  * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
  */
 public class WebSettings {
-    public static ILogger logger = instance("JDI");
-    public static String getDomain() {
-        if (DRIVER.domain != null)
-            return DRIVER.domain;
-        init();
-        return "No Domain Found. Use test.properties or JDISettings.DRIVER.domain";
-    }
-    public static void setDomain(String domain) {
-        DRIVER.domain = domain;
-    }
-    public static VisualCheckAction VISUAL_ACTION_STRATEGY = VisualCheckAction.NONE;
-    public static VisualCheckPage VISUAL_PAGE_STRATEGY = VisualCheckPage.NONE;
-    public static boolean STRICT_SEARCH = true;
-    public static boolean hasDomain() {
-        init();
-        return DRIVER.domain != null && DRIVER.domain.contains("://");
-    }
-    public static String TEST_GROUP = "";
+    private static WebSettings webSettings;
+    private static final JDISettings jdiSettings = getJDISettings();
+
+    public ILogger logger;
+    public VisualCheckAction VISUAL_ACTION_STRATEGY;
+    public VisualCheckPage VISUAL_PAGE_STRATEGY;
+    public boolean STRICT_SEARCH;
+
+    public String TEST_GROUP = "";
     // TODO multi properties example
-    public static Safe<String> TEST_NAME = new Safe<>((String) null);
-    public static String useDriver(JFunc<WebDriver> driver) {
-        return WebDriverFactory.useDriver(driver);
-    }
-    public static String useDriver(String driverName) {
-        return WebDriverFactory.useDriver(driverName);
-    }
-    public static String useDriver(DriverTypes driverType) {
-        return WebDriverFactory.useDriver(driverType);
-    }
-    public static String printSmartLocators(IBaseElement el) {
-        try {
-            return "smart: " + format(ELEMENT.smartTemplate, ELEMENT.smartName.value.execute(el.getName()));
-        } catch (Exception ex) {
-            return format("Can't define smart locator(%s, %s)", ELEMENT.smartTemplate, ELEMENT.smartName.key);
-        }
-    }
+    public Safe<String> TEST_NAME;
+
+    public boolean initialized;
+
     public static JFunc1<IBaseElement, WebElement> SMART_SEARCH = el -> {
-        switch (ELEMENT.useSmartSearch) {
+        switch (jdiSettings.ELEMENT.useSmartSearch) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Switch statements should have a default label](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5482899)